frontend/src/util/routeUtil.js

```diff
@@ -0,0 +1,17 @@
+// import React from "react";
```

Comment:
Remove file?

Reply:
I can remove it. This is for creating protected routes for admin and user, I'll remove it for now